lib/tasks/metrics.rake

```diff
@@ -0,0 +1,26 @@
+require 'csv'
+require 'benchmark'
+
+namespace :metrics   do
+  desc 'Generating # of portfolios per organization'
+  task :portfolios => :environment do
+    Benchmark.bm(15) do |bm|
+      data = [['tenant_id', 'external_tenant', 'count']]
```

Comment:
We'll have lots of columns so they need to have meaningful names.  `count` should be `number_of_portfolios`